src/ocl/dropoutocl.cpp

```diff
@@ -290,25 +290,25 @@ void DropoutDescriptor::DropoutForward(const Handle& handle,
         kernels.front()(pstates,
                         dropout,
                         amp_scale,
-                        int(in_len[1]),
-                        int(in_len[2]),
-                        int(in_len[3]),
-                        int(in_len[4]),
+                        static_cast<int>(in_len[1]),
+                        static_cast<int>(in_len[2]),
+                        static_cast<int>(in_len[3]),
+                        static_cast<int>(in_len[4]),
                         y,
-                        int(out_str[0]),
-                        int(out_str[1]),
-                        int(out_str[2]),
-                        int(out_str[3]),
+                        static_cast<int>(out_str[0]),
+                        static_cast<int>(out_str[1]),
+                        static_cast<int>(out_str[2]),
+                        static_cast<int>(out_str[3]),
                         x,
-                        int(in_str[0]),
-                        int(in_str[1]),
-                        int(in_str[2]),
-                        int(in_str[3]),
+                        static_cast<int>(in_str[0]),
+                        static_cast<int>(in_str[1]),
+                        static_cast<int>(in_str[2]),
+                        static_cast<int>(in_str[3]),
                         reserveSpace,
-                        uint(total_work),
-                        uint(in_offset),
-                        uint(out_offset),
-                        uint(rsvsp_offset));
+                        static_cast<unsigned>(total_work),
```

Comment:
I know `int` is implicit, but shouldn't it be explicit `unsigned int`?

Reply:
@CAHEK7 No, `unsigned` and `unsigned int` are equivalent in Standard C++. [R] mark this "resolved"

Reply:
That's why I've told "implicit".